.github/workflows/step_tests_categories-pip.yml

```diff
@@ -0,0 +1,50 @@
+name: test-categories
```

Comment:
These need to be added to `.github/workflows/ci.yaml`.
Also, remind me to make a PR to re-organize these CI files.